### DISCUSSION TITLE
SET 42176 - New hub

### DISCUSSION THREAD

**Discussion by alpha71** [pybricks/support#1994] [General]
Hello guys, I'd like to know (or maybe i simply missed some topic in the repository :! ) if you have any progress to control the new hub installed in the set 42176 (Porsche GT4) with Pybricks? 
Thank you 
Ale

**Comment by dlech** [comment]
There have been a few discussions about it already.

* https://github.com/pybricks/support/issues/1986
* https://github.com/pybricks/support/issues/1964
* https://github.com/orgs/pybricks/discussions/1733

**Reply by alpha71** [reply]
It means you did not figure out how to make a firmware for this new hub! Then what about the work done by Benedettelli on it?

**Reply by dlech** [reply]
The new hub can still be remote controlled using the LEGO Wireless Protocol.

**Reply by alpha71** [reply]
Ok, then in summary: the pybricks firmware cannot be installed due to the encrypting but it can be remote controlled using the Lego BT protocol with python! That's good in half coz if anyone wants use it in a robot challenge it I'll become difficult (I guess), unless have very good programming skills (as Benedettelli did!) What a pity!
